### PR TITLE
Added sand island model and example world

### PR DESCRIPTION
**Pull request by ahcorde**
Added mcmillan_airfield 
 - model
 - example world

sand_island

co-author: @tfoote

### CODE REVIEW

**Comment by tfoote** [comment]
Please test this out as is for feedback.  Before it's ready to merge we're going to move the models to be hosted at gazebosim.org/models so that the model data isn't stored here in this repository. But before we finalize the model upload there we'd like to get any feedback from reviews here.

**Review by tfoote — APPROVED** [review]
The model is now externally hosted on gazebosim.org and this is just a convenient world for launching from px4. 

Make sure to squash merge to remove the models in the first commit for review.

**Review by tfoote — CHANGES_REQUESTED** [review]
Actually looking at the other models we should add the gps origin here too.

File: worlds/sand_island.world
Comment (Jaeyoung-Lim) [inline worlds/sand_island.world:11]: As tfoote mentioned, can you add the location with the `spherical_coordinate` tag?

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]

File: worlds/sand_island.world
Comment (ahcorde) [reply worlds/sand_island.world:11]: I'm not able to find the original coordinates

Comment (TSC21) [reply worlds/sand_island.world:11]: > I'm not able to find the original coordinates

@tfoote can you help here?

**Comment by Jaeyoung-Lim** [comment]
@ahcorde The origin doesn't seem to be at the same height of the world. This will make the drone fall into the terrain

Screenshot from 2020-03-26 17-13-02

File: worlds/sand_island.world
Comment (tfoote) [reply worlds/sand_island.world:11]: We don't have high precision for this one. The other ones were mostly hand tuned by registering qgc and gazebo visualizations and flying the drones to observable landmarks.

This is a less interesting world and we didn't do the full integration into our demo with it. Which is when I did the high precision registration. Unless there's strong interest in some particular aspect I'd suggest we just close this PR and focus on the other more featureful worlds.

Comment (TSC21) [reply worlds/sand_island.world:11]: What about just making sure that the drone is spawned at the terrain level?

We can focus on cleaning the others but I think this one can be kept as well.

**Comment by ahcorde** [comment]
PR to fix the origin https://bitbucket.org/osrf/gazebo_models/pull-requests/364/fixed-sand_island-coordinates/diff

default_gzclient_camera(1)-2020-03-30T16_28_40 866362

**Comment by Jaeyoung-Lim** [comment]
@ahcorde Thanks! Could you also propagate the change to the `mcmillan_airfield`?

**Comment by ahcorde** [comment]
sure, done ;)

**Comment by Jaeyoung-Lim** [comment]
@ahcorde Thanks! Will merge once the PR is merged in the model database

**Review by Jaeyoung-Lim — APPROVED** [review]

**Comment by TSC21** [comment]
So the spherical coordinates are not added to this one?

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]
@TSC21 Sorry, I was only looking at the model database PR

File: worlds/sand_island.world
Comment (Jaeyoung-Lim) [reply worlds/sand_island.world:11]: @ahcorde @tfoote Are there any updates?

**Comment by Jaeyoung-Lim** [comment]
@tfoote @ahcorde Closing this since this seems stale now. It seems like the models are already in the database. please feel free to reopen when relevant

**Pull request closed without merging**
